config/initializers/doorkeeper_models.rb

```diff
@@ -16,6 +16,8 @@
 
   has_many :security_logs, inverse_of: :application
 
+  delegate :member_ids, to: :owner
```

Comment:
is this being used? if not, use it or else delete it

Reply:
Removed